README.md

```diff
@@ -73,12 +59,27 @@ module.**
 * This module does not come with an entity view page for API products at this moment. (See
   [#72](https://github.com/apigee/apigee-edge-drupal/issues/72).) Unless you have a custom module enabled that provides
   this functionality, do not use the "Rendered entity" field formatter on API product entity reference fields.
+  
+## Applying Patches
 
-## Requirements
+The Apigee Edge module may require Drupal core or contributed module
```

Comment:
I know you have copy-pasted this, but please fix the line-breaks. Use 120 characters.